ddtrace/internal/_encoding.pyx

```diff
@@ -244,6 +244,11 @@ cdef class MsgpackStringTable(StringTable):
     cdef insert(self, object string):
         cdef int ret
 
+        if len(string) > self.max_size:
+            string = "<dropped string of length %d in ddtrace.internal._encoding.pyx due to overrun of buffer of length %d>" % (
```

Comment:
Presumably this string will be visible from the UI. I wonder if we should really add internal details such as `ddtrace.internal._encoding.pyx`. I'd be inclined to keep the message short, e.g. `[dropped due to string being too long]`